src/modules/Layout/Footer/Footer.js

```diff
@@ -5,14 +5,12 @@ import './Footer.css';
 
 const b = block('j-footer');
 
-const Footer = () => {
-  return (
-    <footer className={b}>
-      <p>
-        <strong>Janus</strong> by <a href="http://hellofresh.com">HelloFresh</a>. The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.
-      </p>
-    </footer>
-  );
-};
+const Footer = () => (
+  <footer className={b}>
```

Comment:
this is strange... I've run `--fix` flag... and after that, no indentation errors left...